lib/linked_in/api/v2.rb

```diff
@@ -0,0 +1,91 @@
+module LinkedIn
+  module Api
+
+    # V2 Consumer API
+    #
+    # @see https://docs.microsoft.com/en-us/linkedin/consumer/
+    module V2
+
+      # Obtain profile information for a member.  Currently, the method only
+      # accesses the authenticated user.
+      #
+      # Permissions: r_liteprofile r_emailaddress
+      #
+      # @see https://docs.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/sign-in-with-linkedin?context=linkedin/consumer/context#retrieving-member-profiles
+      #
+      # @return [void]
+      def v2_profile
+        path = '/me'
+        v2_get(path)
+      end
+
+      # Share content for the authenticated user
+      #
+      # Permissions: w_member_share
+      #
+      # @see https://docs.microsoft.com/en-us/linkedin/consumer/integrations/self-serve/share-on-linkedin
+      #
+      # @param [String] urn   User's URN (UID) returned from OAuth access token
+      #                       request
+      # @param [Hash]   share The body we want to submit to LinkedIn. At least a
+      #                       comment is required
+      #
+      # @macro share_input_fields
+      # @return [void]
+      def v2_add_share(urn, share = {})
```

Comment:
Again with the same nitpick. I feel like if it is already scoped to a V2 module it should just be without a prefix on the method.